### PR TITLE
feat: fee asset utility functions

### PR DESCRIPTION
**Pull request by 0xdef1cafe**
## Description

how do we not already have these? handy util functions to get fee asset id from chain or asset ids

## Notice

- [x] Have you followed the guidelines in our [Contributing]('https://github.com/shapeshift/web/blob/develop/CONTRIBUTING.md) guide?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/shapeshift/web/pulls) for the same update/change?

## Pull Request Type

- [ ] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [ ] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [x] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)

n/a - developer QOL

## Risk

low - not consumed anywhere, surprised we don't have these already

### Engineering

note - this non-null assertion *is* going to fuck us when we turn osmo on - we'll need to do a redux persistence migration to blow away persisted account ids in the store if we ever turn it off, e.g. there will exist account ids we no longer have a chain adapter for

### Operations

testing not required by ops

## Screenshots (if applicable)

n/a

### CODE REVIEW

**Review by gomesalexandre — APPROVED** [review]
🧠 

Untested but pretty straightforward and pure helpers, LGTM

**Review by 0xApotheosis — APPROVED** [review]
Non-blocking commentary.

File: src/state/slices/assetsSlice/selectors.ts [hunk 1]
```diff
@@ -25,6 +25,7 @@ import { selectCryptoMarketDataIdsSortedByMarketCap } from 'state/slices/marketD
 export const selectAssetById = createCachedSelector(
   (state: ReduxState) => state.assets.byId,
   (_state: ReduxState, assetId: AssetId) => assetId,
+  // TODO(0xdef1cafe): make this return type AssetId | undefined and fix the 600+ type errors
```
[the remark this answers is not in the document]
Comment (0xdef1cafe) [reply src/state/slices/assetsSlice/selectors.ts:28]: it hurts my heart

File: src/pages/Accounts/components/AccountNumberRow.tsx [hunk 2]
```diff
@@ -135,7 +135,7 @@ export const AccountNumberRow: React.FC<AccountNumberRowProps> = ({
   const title = useMemo(
     () =>
       isUtxoAccount
-        ? assets[accountIdToFeeAssetId(accountId)].name
+        ? assets[accountIdToFeeAssetId(accountId ?? '') ?? '']?.name ?? ''
         : firstFourLastFour(fromAccountId(accountId).account),
     [assets, accountId, isUtxoAccount],
   )
```
[the remark this answers is not in the document]
Comment (0xdef1cafe) [reply src/pages/Accounts/components/AccountNumberRow.tsx:141]: agreed

**Review by 0xApotheosis — APPROVED** [review]
🥧

**Pull request merged**